src/app/components/buttons/BigButton/BigButton.scss

```diff
@@ -0,0 +1,17 @@
+@import '@/styles/base/variables';
+
+.big-button {
```

Comment:
big-button class seems a little bit suspisous.

I can see there is BigButton component.
It was better to create Button component and then via props for example
type={"big"} send this BigButton styling, but it would be via classname change of Button component so it will be like className={"button",{`${type}`}

Creating BigButton, SmallButton, Button components its a bit not good practice